test/verify/check-machines

```diff
@@ -313,13 +313,6 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b.wait_present("#chart-donut-1 .donut-title-big-pf")
         self.assertLessEqual(get_usage("#chart-donut-1"), 100.0)
 
-        # suspend/resume
-        m.execute("virsh suspend subVmTest1")
-        b.wait_in_text("#vm-subVmTest1-state", "paused")
-        # resume sometimes fails with "unable to execute QEMU command 'cont': Resetting the Virtual Machine is required"
-        m.execute('virsh resume subVmTest1 || { virsh destroy subVmTest1 && virsh start subVmTest1; }')
-        b.wait_in_text("#vm-subVmTest1-state", "running")
```

Comment:
But that does something different - it makes sure that the page picks up state changes from "somewhere else", i. e. not the ones it triggered itself. I think this is useful, and also quite cheap. Let's please keep this?

Reply:
Dropped the commit that removed it